lib/cinemavision/actions.py

```diff
@@ -99,10 +107,9 @@ def getStartupInfo():
 class ScriptCommand(SubprocessActionCommand):
     type = 'SCRIPT'
 
-    def execute(self):
-        command = ['python', self.commandData]
+    def execute(self):        
+        command = [self._absolutizeCommand()]
```

Comment:
I think this needs to be changed to `command = ['python', self._absolutizeCommand()]` and then I can merge this.